packages/web-components/@sfx/sayt/tsconfig.json

```diff
@@ -7,11 +7,14 @@
       "dom"
     ],
     "types": [
-      "node"
+      "node",
+      "mocha",
+      "chai",
+      "sinon"
     ],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "ES5",
+    "target": "es2015",
```

Comment:
I'm not too familiar with `tsconfig.json`, but is this intentional? Why are we no longer targeting ES5?

Reply:
As discussed, we'll be keeping this as `es2015` for now and will look to incorporate polyfills in the future.